OpenDreamClient/Interface/DMF/DMFParser.cs

```diff
@@ -203,12 +203,19 @@ public sealed class DMFParser : Parser<char> {
             }
 
             Token attributeValue = Current();
-            if (!Check(TokenType.DMF_Value))
-                Error($"Invalid attribute value ({attributeValue.Text})");
+            if (!Check(TokenType.DMF_Value)) {
```

Comment:
I'm not sure whether the changes to this file work as intended.

Reply:
A value of `none` leads to the attribute being ignored entirely. This would work if it was the initial value in a DMF, though it'll get ignored in a winset. I'm actually not sure how this would be fixed given how we use ISerializationManager.

Should also note a newline does the same thing as a semicolon here.